server/client.go

```diff
@@ -2043,15 +2058,26 @@ func (c *client) processInboundClientMsg(msg []byte) {
 		c.checkForImportServices(c.acc, msg)
 	}
 
+	var queuesa [16][]byte
```

Comment:
If tmp maybe just qa since queues is what we reference in code.

Reply:
done